structuredefinitions/UKCore-DiagnosticReport.xml

```diff
@@ -149,6 +131,25 @@
       <short value="Observations that are part of this diagnostic report." />
       <mustSupport value="true" />
     </element>
+    <element id="DiagnosticReport.media">
+      <path value="DiagnosticReport.media" />
+      <constraint>
+        <key value="ukcore-diag-lab-002" />
+        <severity value="error" />
+        <human
+          value="Only one of DiagnosticReport.media.link or DiagnosticReport.media:mediaLinkR5 can be used at a time." />
+        <expression value="media.where(sliceName.exists()).count() = 0 or media.link.exists() = false" />
```

Comment:
There isn't a slice anymore. Can you follow https://github.com/NHSDigital/FHIR-R4-UKCORE-STAGING-MAIN/blob/fad692ab52970801afb9bbbf90e260fe8b57fdb1/structuredefinitions/UKCore-Specimen.xml#L35C9-L36C199:

```<human value="There SHALL be only one reference between `collection.collector` and the extension collectionCollectorR5`" />```
```<expression value="collection.collector.reference.empty() or collection.collector.extension('http://hl7.org/fhir/5.0/StructureDefinition/extension-Specimen.collection.collector').empty()" />```